<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-KFM2KDI1" failures="1" tests="8" name="com.supermarket.tests.ManageDeliveryBoyTest" time="78.619" errors="0" timestamp="2022-11-10T19:00:45 IST" skipped="0">
  <testcase classname="com.supermarket.tests.ManageDeliveryBoyTest" name="verify_search_byName" time="7.079"/>
  <system-out/>
  <testcase classname="com.supermarket.tests.ManageDeliveryBoyTest" name="verify_edit_option" time="10.457"/>
  <system-out/>
  <testcase classname="com.supermarket.tests.ManageDeliveryBoyTest" name="verify_search_byEmail" time="7.461"/>
  <system-out/>
  <testcase classname="com.supermarket.tests.ManageDeliveryBoyTest" name="verify_status" time="9.885"/>
  <system-out/>
  <testcase classname="com.supermarket.tests.ManageDeliveryBoyTest" name="verify_search_byPhonenumber" time="7.691"/>
  <system-out/>
  <testcase classname="com.supermarket.tests.ManageDeliveryBoyTest" name="verify_new_deliveryBoy_creation" time="11.717">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.supermarket.tests.ManageDeliveryBoyTest.verify_new_deliveryBoy_creation(ManageDeliveryBoyTest.java:154)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- verify_new_deliveryBoy_creation -->
  <system-out/>
  <testcase classname="com.supermarket.tests.ManageDeliveryBoyTest" name="verify_deleteButton_editOption" time="8.433"/>
  <system-out/>
  <testcase classname="com.supermarket.tests.ManageDeliveryBoyTest" name="verify_create_delivery_boy" time="15.896"/>
  <system-out/>
</testsuite> <!-- com.supermarket.tests.ManageDeliveryBoyTest -->
